src/main/java/com/extrahardmode/config/RootNode.java

```diff
@@ -53,6 +53,22 @@
      */
     PRINT_COMMENTS("Print Comments", VarType.BOOLEAN, true, "If comments like this should be printed"),
     /**
+    * Globally disables certain modules. These settings might improve performance if these modules is not used. 
```

Comment:
No if you dont disable these modules they will run and check if they are active every time. Feel free to see for yourself in spigot timing. E.g torch module will check every time you place a torch if it should cancel it out. Might not sound like allot, but when it scales up it becomes quite a bit. EHM was one of my most demanding plugin in timings even if I did not use the features that was most resource heavy. After doing this I have to scroll way down to see EHM in timings.